homeassistant/components/mystrom/switch.py

```diff
@@ -73,36 +77,37 @@ class MyStromSwitch(SwitchEntity):
     _attr_has_entity_name = True
     _attr_name = None
 
-    def __init__(self, plug, name):
+    def __init__(self, device, name):
```

Comment:
Please do these changes in a separate PR

Reply:
understandable, I've reverted this changes